pkg/runtime/from_config.go

```diff
@@ -15,6 +16,24 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	crdRegex  = "^core.openfeature.dev/"
+	fileRegex = "^file:"
+	urlRegex  = "^https?://"
+)
+
+var (
+	regCrd  *regexp.Regexp
+	regURL  *regexp.Regexp
+	regFile *regexp.Regexp
+)
+
+func init() {
+	regCrd = regexp.MustCompile(crdRegex)
```

Comment:
Nitpicking but it might suffice to just use the regex strings here directly rather than constants, I'll leave the decision to you